.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature Request
+about: Request a feature addition or change to Capacitor.
+title: 'feat: '
+labels: ''
+assignees: ''
+
+---
+
+# Feature Request
+
+## Describe the Feature Request
+<!-- A clear and concise description of what the feature request is. Please include if your feature request is related to a problem. -->
+
+## Platform Support Requested
+<!-- Check one or more boxes for the platform(s) you would like this feature to support. Note that some features may not be feasible for a particular platform (i.e. no web API may be available). -->
```

Comment:
Should be similar to the bug note (`<!-- Leave these checkboxes EMPTY until after the issue is created. Once the issue is created, check one or more boxes for the platform(s) this bug affects. -->`)

Reply:
Fixed :)